example/py-shiny/example_pyshiny_reactive_scatter.py

```diff
@@ -0,0 +1,57 @@
+from maidr.custom_renderers.shiny_renderer import render_maidr
+import matplotlib.pyplot as plt
+import pandas as pd
```

Comment:
Why are you importing `pandas`? Is this used?

Reply:
`pandas` was used with the older example but I did not have that removed for the new scatterplot. It has been removed in the latest commit.